enos/modules/backend_test_servers/scripts/setup_openldap.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+set -e
+
+fail() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+[[ -z "$LDAP_DOMAIN" ]] && fail "LDAP_DOMAIN env variable has not been set"
+[[ -z "$LDAP_ORG" ]] && fail "LDAP_ORG env variable has not been set"
+[[ -z "$LDAP_ADMIN_PW" ]] && fail "LDAP_ADMIN_PW env variable has not been set"
+[[ -z "$LDAP_VERSION" ]] && fail "LDAP_VERSION env variable has not been set"
```

Comment:
Hrm. This the container version, not the OpenLDAP version. Perhaps we ought to make that distinction?